app/src/app/logToFile.js

```diff
@@ -13,11 +13,11 @@ function escapePath(path) {
   return path.replace(/( |\(|\))/g, `${escapeChar}$1`)
 }
 
-function hookStream(stream, callback) {
-  const {write} = stream
-  stream.write = function() {
+function hookStream(src, method, callback) {
```

Comment:
I would rename it to `wrap` or something similar, because in case of console.log its  not a stream any more.